cli/commands/benchmark-web-vitals.mjs

```diff
@@ -131,6 +131,17 @@ async function benchmarkURL( browser, params ) {
 		},
 	};
 
+	/*
+	 * Aggregate metrics are metrics which are calculated for every request as
+	 * a combination of other metrics.
+	 */
+	const aggregateMetricsDefinition = {
+		'LCP - TTFB': {
```

Comment:
This might be more clear:

```suggestion
		'LCP w/o TTFB': {
```

Otherwise it could seem like it's reporting the TTFB part of LCP, when actually it means LCP without TTFB.

Reply:
@westonruter Fair point. I find the "w/o" a bit odd though, not necessarily an acronym everyone is familiar with. How about "LCP-TTFB"? Maybe the lack of space makes it clear that it's actually a "minus" sign.

Reply:
Oh, I thought it would be commonly understood to mean "without". Otherwise, "LCP-TTFB" I think is a bit better.

Reply:
Updated in https://github.com/GoogleChromeLabs/wpp-research/pull/50/commits/1723bcf799a8a67a8ab087bc8ff8a90a0355b29f.